src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -141,6 +142,18 @@ public override void VisitMarkupBlock(MarkupBlockSyntax node)
 
         public override void VisitMarkupElement(MarkupElementSyntax node)
```

Comment:
Do we not need to do the same thing for every block type? Wondering if there's a better hook we can tie into here like `VisitRazorDirectiveBody`

Reply:
ugh, great idea.. i could probably roll this into the CSharpBlock handling.. maybe there is a method to the madness of the syntax tree structure after all :D